app/components-react/highlighter/ExportModal.tsx

```diff
@@ -220,84 +244,212 @@ function ExportOptions({
-          label={$t('Video Name')}
-          value={videoName}
-          onInput={name => {
-            onVideoNameChange(name);
-            setExportFile(getExportFileFromVideoName(name));
-          }}
-          uncontrolled={false}
-        />
-        <FileInput
-          label={$t('Export Location')}
-          name="exportLocation"
-          save={true}
-          filters={[{ name: $t('MP4 Video File'), extensions: ['mp4'] }]}
-          value={exportFile}
-          onChange={file => {
-            setExportFile(file);
-            onVideoNameChange(getVideoNameFromExportFile(file));
-          }}
-        />
-        <RadioInput
-          label={$t('Resolution')}
-          value={exportInfo.resolution.toString()}
-          options={[
-            { value: '720', label: '720p' },
-            { value: '1080', label: '1080p' },
-          ]}
-          onChange={setResolution}
-          buttons={true}
-        />
-        <RadioInput
-          label={$t('Frame Rate')}
-          value={exportInfo.fps.toString()}
-          options={[
-            { value: '30', label: '30 FPS' },
-            { value: '60', label: '60 FPS' },
-          ]}
-          onChange={setFps}
-          buttons={true}
-        />
-        <RadioInput
-          label={$t('File Size')}
-          value={exportInfo.preset}
-          options={[
-            { value: 'ultrafast', label: $t('Faster Export') },
-            { value: 'fast', label: $t('Balanced') },
-            { value: 'slow', label: $t('Smaller File') },
-          ]}
-          onChange={setPreset}
-          buttons={true}
-        />
-        {exportInfo.error && (
-          <Alert
-            style={{ marginBottom: 24 }}
-            message={exportInfo.error}
-            type="error"
-            closable
-            showIcon
-            afterClose={dismissError}
-          />
-        )}
-        <div style={{ textAlign: 'right' }}>
-          <Button style={{ marginRight: 8 }} onClick={close}>
-            {$t('Close')}
-          </Button>
-
-          <Button style={{ marginRight: 8 }} type="primary" onClick={() => startExport('vertical')}>
-            {$t('Export Vertical')}
-          </Button>
-          <Button type="primary" onClick={() => startExport('horizontal')}>
-            {$t('Export Horizontal')}
-          </Button>
+    <Form>
+      <div className={styles.modalWrapper}>
+        <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+          <h2 style={{ fontWeight: 600, margin: 0 }}>{$t('Export')}</h2>{' '}
+          <div>
+            <Button type="text" onClick={close}>
+              <i className="icon-close" style={{ margin: 0 }}></i>
+            </Button>
+          </div>
         </div>
-      </Form>
-    </div>
+        <div style={{ display: 'flex', gap: '16px' }}>
+          <div className={styles.settingsAndProgress}>
+            <div className={`${styles.pathWrapper} ${isExporting ? styles.isDisabled : ''}`}>
```

Comment:
we use a library `classnames` to help with dynamic css classes like this, you can search `cx(` in our codebase to see some examples